src/labels/CustomLabels.labels

```diff
@@ -1732,7 +1734,13 @@ If you&apos;re not sure how to resolve these errors, post a message in the Nonpr
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Message displayed on a successful migration</shortDescription>
-        <value>Your migration was successful with no errors. All set up steps are now complete. You're ready to begin using Enhanced Recurring Donations.</value>
+        <value>You're now finished with the Enhanced Recurring Donations upgrade process. You've enabled the new functionality, installed metadata, and migrated your existing data.
```

Comment:
The message was not displayed, the issue is fixed now. Can you please verify?

Reply:
@RadSFDeveloper - Yes, I see the message now, thank you.